arenamanager.go

```diff
@@ -10,17 +10,18 @@ import (
 )
 
 const (
-	cArenaFilePrefix = "_arena.dat"
+	cArenaFileSuffix = "_arena.dat"
 )
 
 // arenaManager manages all the arenas for a bigqueue
 type arenaManager struct {
-	dir     string
-	conf    *bqConfig
-	md      *metadata
-	baseAid int
-	arenas  []*mmap.File
-	inMem   int
+	dir       string
+	conf      *bqConfig
+	md        *metadata
+	baseAid   int
+	arenas    []*mmap.File
+	inMem     int
+	filePath  []byte
```

Comment:
Could we call it something other than `filePath`. This happens to be a pkg name in Go.

Reply:
Indeed, how about fullPath?